### PR TITLE
Add missing project file

### PR DESCRIPTION
**Pull request by ColinSullivan1**
Add missing Publish project file.

Signed-off-by: Colin Sullivan <colin@synadia.com>

### CODE REVIEW

File: examples/Publish/Publish.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>NATS Publish Example</Description>
+    <Copyright>Copyright The NATs Authors 2016-2018</Copyright>
+    <AssemblyTitle>NATS Replier</AssemblyTitle>
```
Comment (kozlovic) [inline examples/Publish/Publish.csproj]: `NATS Publisher` instead?

**Review by kozlovic — APPROVED** [review]
Other than what look like a wrong assembly title, LGTM

File: examples/Publish/Publish.csproj (hunk 1 above)
Comment (ColinSullivan1) [reply examples/Publish/Publish.csproj]: D'OH.  Thanks!  Will Fix.

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**